test/new-e2e/pkg/components/operator.go

```diff
@@ -0,0 +1,15 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package components
+
+import (
+	"github.com/DataDog/test-infra-definitions/components/datadog/operator"
+)
+
+// Operator is a Datadog Operator running in a Kubernetes cluster
+type Operator struct { // nolint:revive, We want to keep the name as <Component>
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Is lint complaining here? It should not, as the package's name is not `operator`
```suggestion
type Operator struct {
```

Reply:
Ah thanks, that definitely shouldn't be there